content/docs/1.0.x/install/authenticate-cli-bridge/index.md

```diff
@@ -7,113 +7,123 @@ weight: 60
 To authenticate and associate the Keptn CLI and Keptn bridge to your Keptn cluster,
 you need the exposed Keptn endpoint and API token.
 The endpoint values and the method of querying them
-is determined by the access options you use.
+are determined by the access options you use.
 See [Choose access options](../access) for details.
 
-## Authenticate Keptn CLI
+## Get the Keptn endpoint
 
-To authenticate the Keptn CLI against the Keptn cluster, the exposed Keptn endpoint and API token are required. 
+To authenticate the Keptn CLI against the Keptn cluster,
+the exposed Keptn endpoint and API token are required. 
 After [installing Keptn](../helm-install), you already have your Keptn endpoint.
 
-<details><summary>Get API Token and Authenticate Keptn CLI on **Linux / MacOS**</summary>
-<p>
 
-* Set the environment variable `KEPTN_API_TOKEN`:
+Get the Keptn endpoint from the `api-gateway-nginx`.
+If you are using port-forward to expose Keptn,
+your endpoint is `localhost` and the `port` to which you forwarded Keptn
+such as `http://localhost:8080`.)
 
-```console
-KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
-```
+   ```
+   kubectl -n keptn get service api-gateway-nginx
+   ```
 
-* To authenticate the CLI against the Keptn cluster, use the [keptn auth](../../reference/cli/commands/keptn_auth) command:
+   ```
+   NAME                TYPE        CLUSTER-IP    EXTERNAL-IP                  PORT(S)   AGE
+   api-gateway-nginx   ClusterIP   10.117.0.20   <ENDPOINT_OF_API_GATEWAY>    80/TCP    44m
```

Comment:
The `EXTERNAL_IP` field will only show the endpoint/IP if the service is set to type `LoadBalancer`. In the port forwarding case this field is not  set at all.

Reply:
Oh ugh!  This all needs to be rewritten but not today.  So I put a link to the access page for details about how to get the endpoint for each access option and made this a "For example, if you are using LoadBalancer..."  See if this works for now.